src/linux/BTSerialPortBindingServer.cc

```diff
@@ -345,30 +330,26 @@ void BTSerialPortBindingServer::Init(Handle<Object> target) {
     Nan::SetPrototypeMethod(t, "write", Write);
     Nan::SetPrototypeMethod(t, "read", Read);
     Nan::SetPrototypeMethod(t, "close", Close);
+    Nan::SetPrototypeMethod(t, "disconnectClient", DisconnectClient);
+    Nan::SetPrototypeMethod(t, "isOpen", IsOpen);
 
     target->Set(Nan::New("BTSerialPortBindingServer").ToLocalChecked(), t->GetFunction());
-    target->Set(Nan::New("BTSerialPortBindingServer").ToLocalChecked(), t->GetFunction());
-    target->Set(Nan::New("BTSerialPortBindingServer").ToLocalChecked(), t->GetFunction());
```

Comment:
Any idea why this line was duplicated in the original?

Reply:
No, the only thing I can think of was that there were 3 exposed methods before, and there's code right above it that had to be called once per method, so maybe this was a copy/paste error where the coder thought this needed to be per exposed method but never went back to try to change it (and then realized it wasn't specific per method)...